app/qml/RecordToolbar.qml

```diff
@@ -48,30 +55,61 @@ Item {
         }
 
         Item {
-            width: parent.width/parent.children.length
+            Layout.fillWidth: true
+            height: parent.height
+            visible: root.pointLayerSelected ? false : true
+
+            MainPanelButton {
+                id: removePointButton
+                width: root.itemSize
+                text: qsTr("Delete")
+                imageSource: "trash.svg"
```

Comment:
I would prefer to call this "Undo Point" and have the undo icon there as well (delete makes me think it would delete the whole feature)